demosplan/DemosPlanCoreBundle/Logic/Faq/FaqService.php

```diff
@@ -20,12 +20,13 @@
 use demosplan\DemosPlanCoreBundle\Entity\User\Role;
 use demosplan\DemosPlanCoreBundle\Entity\User\User;
 use demosplan\DemosPlanCoreBundle\Exception\CustomerNotFoundException;
-use demosplan\DemosPlanCoreBundle\Logic\ApiRequest\EntityFetcher;
```

Comment:
It seems, the entityFetcher is still needed here?!
`return $this->entityFetcher->listEntitiesUnrestricted($className, $conditions, [$sortMethod]);`
in line 168